web/base_pdf_page_view.js

```diff
@@ -152,10 +193,17 @@ class BasePDFPageView {
     canvas.remove();
     canvas.width = canvas.height = 0;
     this.canvas = null;
+    if (this.#tempCanvas) {
+      this.#tempCanvas.width = this.#tempCanvas.height = 0;
+      this.#tempCanvas = null;
+    }
   }
 
   async _drawCanvas(options, onCancel, onFinish) {
     const renderTask = (this.renderTask = this.pdfPage.render(options));
+    if (this.pdfPage._pdfBug && globalThis.StepperManager?.enabled) {
+      this.#minDurationToUpdateCanvas = 0;
+    }
```

Comment:
Do we need to check this here, can't we set the option together with the others when loading the debugger?
https://github.com/mozilla/pdf.js/blob/efc5c3c2314840098551c325cd7e31d60ef3bd53/web/app.js#L327-L337